ngraph/cmake/coverage.cmake

```diff
@@ -22,7 +22,7 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     ie_coverage_genhtml(INFO_FILE "onnx_importer"
         PREFIX "${NGRAPH_COVERAGE_BASE_DIRECTORY}")
 endif()
-if (NGRAPH_ONNX_EDITOR_ENABLE)
+if (NGRAPH_ONNX_IMPORT_ENABLE)
```

Comment:
maybe we can reuse
https://github.com/openvinotoolkit/openvino/blob/5765a7e35c27102d92eb1f84d982e02d2b574ba7/cmake/developer_package/options.cmake#L14-L17
to make dependencies between `NGRAPH_ONNX_EDITOR_ENABLE` and `NGRAPH_ONNX_IMPORT_ENABLE`

Looks really strange that `NGRAPH_ONNX_IMPORT_ENABLE` defines stuff for `onnx_editor`

Reply:
@ilya-lavrenov I've pushed this change only to check if everything will be OK if I change CI configuration to build and run `onnx_editor` tests on all configurations where `onnx_importer` is run